### PR TITLE
Adding a page with general info about AlmaLinux Build System

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

File: docs/development/AlmaLinux-Build-System.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# AlmaLinux Build System
+
+## About AlmaLinux Build System
+
+[AlmaLinux Build System](https://build.almalinux.org/) is designed to handle processes of:
+* Building rpm packages for multiple supported architectures and distribution versions.
+* Testing packages.
+* Signing packages. See more on the [Signing packages](/development/private-keys/package-signing) page.
+* Releasing packages.
+* Ensuring that packages and their sources are reliable using [SBOM](/documentation/sbom-guide).
```
Comment (bennyvasquez) [inline docs/development/AlmaLinux-Build-System.md:16]:
```suggestion
* Ensuring that packages and their sources are reliably using [SBOM](/documentation/sbom-guide).
```

File: docs/development/AlmaLinux-Build-System.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* Providing [Errata](/documentation/errata).
+
+## AlmaLinux OS source packages 
+
+AlmaLinux OS source code including [modified packages](/development/Modified-packages) is stored and managed in Git repositories on [git.almalinux.org](https://git.almalinux.org/explore/repos).
+
+AlmaLinux OS Team and Community work on these repositories to submit changes, fix bugs, and introduce new features. The AlmaLinux Build System is used then to pull the latest changes from these repositories and build updated packages.
```
Comment (bennyvasquez) [inline docs/development/AlmaLinux-Build-System.md:23]:
```suggestion
AlmaLinux OS Team and community work on these repositories to submit changes, fix bugs, and introduce new features. The AlmaLinux Build System is then used to pull the latest changes from these repositories and build the updated packages. 
```

File: docs/development/AlmaLinux-Build-System.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## More guides and details
+
+If you are interested in learning how to build AlmaLinux OS packages, please check the [Packaging guidelines](/development/Packaging) and [Building packages guide](/documentation/building-packages-guide).
+
+Details and guides on how the AlmaLinux Build System works and how to use it can be found in the [AlmaLinux Build System Wiki](https://github.com/AlmaLinux/build-system/wiki).
+
+## Reporting Issues
+
+All issues should be reported to the [Build System](https://github.com/AlmaLinux/build-system/issues) repository. A [GitHub's kanban board](https://github.com/orgs/AlmaLinux/projects/2) is used for the Build System project to track the progress.
```
Comment (bennyvasquez) [inline docs/development/AlmaLinux-Build-System.md:33]:
```suggestion
All issues should be reported to the [Build System](https://github.com/AlmaLinux/build-system/issues) repository. A [kanban board](https://github.com/orgs/AlmaLinux/projects/2) is used for the Build System project to track the progress.
```

**Review by bennyvasquez — APPROVED** [review]
just a couple small edits for English. Other than that, this looks great!

**Comment by sboldyreva** [comment]
@bennyvasquez thanks!

**Pull request merged**